discovery-provider/alembic/trigger_sql/handle_track.sql

```diff
@@ -47,7 +47,7 @@ begin
           subscriber_user_ids,
           new.updated_at,
           'create',
-          new.owner_id,
+          new.track_id,
```

Comment:
will we have to fix any existing notifs to have the correct specifier? not sure if we query by specifier anywhere and whether this would break it